src/views/start.njk

```diff
@@ -1,18 +1,114 @@
 {% extends "_base.njk" %}
 
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/accordion/macro.njk" import govukAccordion %}
 {% block content %}
   <h1 class="govuk-heading-xl">
-    Fit and Competent
+     Fit and Competent Controllers Registration
   </h1>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
+
+      <p class="govuk-body">
+        You must hold Fit and Competent Controller status if you wish to control deer:
+      </p>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>
+          Out of season (under a specific <a class="govuk-link" rel="noreferrer noopener" target="_blank" href="http://gateway.snh.gov.uk/go/?id=35">5(6)</a> authorisation of the Deer Act)
+        </li>
+        <li>
+          At night (under the <a class="govuk-link" rel="noreferrer noopener" target="_blank" href="http://gateway.snh.gov.uk/go/?id=36">18(2)</a> authorisation of the Deer Act)
+        </li>
+        <li>
+          Under the <a class="govuk-link" rel="noreferrer noopener" target="_blank" href="http://gateway.snh.gov.uk/go/?id=34">general authorisation</a> (if you don’t meet the criteria below)
+        </li>
+      </ul>
+
+      <p class="govuk-body">
+        ! You can use this service to apply, amend, renew or surrender your status
+      </p>
```

Comment:
This should be a [GOV.UK Warning Text component](https://design-system.service.gov.uk/components/warning-text/). That's what they mean if you see a line beginning with a `!` in an issue.

See [start.njk](https://github.com/Scottish-Natural-Heritage/trap-registration/blob/main/src/views/start.njk#L34-L39) in trap-registration.

Reply:
Resolved in c452157362c04c6d2a93f2e9c4e46915d91d02eb.